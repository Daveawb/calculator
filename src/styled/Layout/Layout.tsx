import styled from 'styled-components';

export const Layout = styled.div`
    display: flex;
    flex-direction: column;
    margin-left: auto;
    margin-right: auto;
    background-color: #1795d4;
    border-radius: 1rem;
    padding: 1rem;
    box-shadow: 1px 1px rgba(0.5, 0, 0, 0);
`;
