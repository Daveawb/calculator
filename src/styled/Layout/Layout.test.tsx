import { shallow } from 'enzyme';
import * as React from 'react';
import { Layout } from './Layout';

describe('<Layout />', () => {
    test('matches snapshot', () => {
        const wrapper = shallow(<Layout />);
        expect(wrapper).toMatchSnapshot();
    });
});
