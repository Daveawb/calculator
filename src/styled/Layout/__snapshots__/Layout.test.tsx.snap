// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Layout /> matches snapshot 1`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-bdVaJa",
        "isStatic": true,
        "rules": Array [
          "
    display: flex;
    flex-direction: column;
    margin-left: auto;
    margin-right: auto;
    background-color: #1795d4;
    border-radius: 1rem;
    padding: 1rem;
    box-shadow: 1px 1px rgba(0.5, 0, 0, 0);
",
        ],
      },
      "displayName": "styled.div",
      "foldedComponentIds": Array [],
      "render": [Function],
      "self": [Circular],
      "styledComponentId": "sc-bdVaJa",
      "target": "div",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
/>
`;
